bin/publish-stories.js

```diff
@@ -0,0 +1,65 @@
+const request = require('request');
+
+const existingComments = (accountName, accountKey, issueNumber) => new Promise((resolve, reject) => {
+
+    const commentsToDelete = [];
+    const auth = 'Basic ' + new Buffer(accountName + ':' + accountKey).toString('base64');
+    const options = {
+      url: `https://api.github.com/repos/newsuk/times-components/issues/${issueNumber}/comments`,
+      headers: {
+       'Authorization': auth,
+       'User-Agent': accountName
+      }
+    };
+
+    request.get(options, function(error, response, body) {
+        if (error) reject(error);
+        JSON.parse(body).forEach((comment) => {
+          if(comment.user.login === accountName) {
+            commentsToDelete.push(comment.id);
```

Comment:
this could be written without mutation:
```
const ids = JSON.parse(body)
  .filter( ({user}) => user.login === accountName)
  .map( ({id}) => id);
  resolve(ids);
```